Examples/buildAndTest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+declare -a repositories=("aspnetcore" "owin")
+
+for i in "${repositories[@]}"; do
+    if [ -d $i ]; then
+	pushd $i
+	./buildAndTest.sh
+        if [ $? -ne 0 ]; then
```

Comment:
Ypu're right, we definetly need to do something about the build system. For example, the script is duplicated in many places